sources/api/netdog/src/cli/mod.rs

```diff
@@ -165,6 +167,46 @@ where
     Ok(())
 }
 
+// Gather net config from possible sources, returning both the config and the source
+fn fetch_net_config() -> Result<(Option<Box<dyn Interfaces>>, PathBuf)> {
+    let over_ride = PathBuf::from(OVERRIDE_NET_CONFIG_FILE);
+    let default = PathBuf::from(DEFAULT_NET_CONFIG_FILE);
```

Comment:
```suggestion
    let override_path = PathBuf::from(OVERRIDE_NET_CONFIG_FILE);
    let default_path = PathBuf::from(DEFAULT_NET_CONFIG_FILE);
```

`default` is also getting keyword highlighting in this view